internal/config/profile.go

```diff
@@ -55,10 +55,15 @@ const (
 	output                       = "output"
 	fileFlags                    = os.O_CREATE | os.O_TRUNC | os.O_WRONLY
 	configPerm                   = 0600
+	skipUpdateCheck              = "skip_update_check"
```

Comment:
@fmenezes Will this also be available as an environment variable?

Reply:
yes MCLI_SKIP_UPDATE_CHECK

Reply:
Ok, added that to the Docs Summary in the ticket